doc/whatsnew/2.9.rst

```diff
@@ -15,6 +15,8 @@ New checkers
 * ``consider-using-dict-items``: Emitted when iterating over dictionary keys and then
   indexing the same dictionary with the key within loop body.
 
+* Add ignore_signatures to duplicate code checker
```

Comment:
```suggestion
* An `ignore_signatures` option has been added to the similarity checker. It will permits  to reduce false positives when multiple functions have the same parameters.
```